frontends/web/src/routes/buy/exchange.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright 2022 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { translate, TranslateProps } from '../../decorators/translate';
+import { FunctionComponent } from 'react';
```

Comment:
we don't use that anymore afaik, see https://github.com/digitalbitbox/bitbox-wallet-app/pull/1839

Reply:
Nice! Fixed